modules/programs/dircolors.nix

```diff
@@ -17,6 +17,11 @@ in {
       '';
     };
 
+    package = lib.mkPackageOption pkgs "dircolors" {
+      default = "coreutils";
+      nullable = true;
```

Comment:
Can't be nullable, it's used in the integration options.

Reply:
Oops thanks